lib/streamlit/elements/arrow.py

```diff
@@ -657,6 +666,19 @@ def table(self, data: Data = None) -> DeltaGenerator:
            https://doc-table.streamlit.app/
            height: 480px
 
+
+        >>> df = pd.DataFrame(
+        ...     {
+        ...         "Command": ["**st.table**", "*st.dataframe*"],
+        ...         "Type": ["`static`", "`interactive`"],
+        ...         "URL": ["[docs](https://docs.streamlit.io)", "🔗"],
+        ...     }
+        ... )
+        >>> st.table(df)
+
+        .. output::
+           https://doc-table-markdown.streamlit.app/
```

Comment:
@sfc-gh-dmatthews we still need to deploy an app here or integrate the code into the previous app. Also, feel free to change the code in any way you want!